lexers/g/groff.go

```diff
@@ -0,0 +1,42 @@
+package g
+
+import (
+	"regexp"
+	"unicode"
+
+	. "github.com/alecthomas/chroma" // nolint
+	"github.com/alecthomas/chroma/lexers/internal"
+)
+
+var groffAlphanumericRe = regexp.MustCompile(`^[a-zA-Z0-9.]+$`)
```

Comment:
We got to remove the dot from the regex pattern here.
```suggestion
var groffAlphanumericRe = regexp.MustCompile(`^[a-zA-Z0-9]+$`)
```

Reply:
it was by mistake.